website/sidebars.js

```diff
@@ -49,8 +57,8 @@ module.exports = {
         },
       ],
       Advanced: ['overview/beanstalk/beanstalk'],
+      Tutorials: tutorials(),
     },
-    'overview/tutorials/tutorials',
```

Comment:
And then you can swap this line out with
```jsx
fbContent({ internal: 'overview/tutorials/tutorials', external: { Tutorials: tutorials() } }),
```
this will allow our internal audience to view the tutorials overview page as well as rendering the tutorials for the external readers :). Thank you!